README.md

```diff
@@ -4,19 +4,145 @@ A repository that serves as a template for deploying small pipelines and
 Cloud Functions in GCP.
 
 This template uses Python for implementing all the logic deployed in your GCP 
-Functions and Terraform as a deployment technique.
+Functions and Terraform as a deployment system.
 
 
-### HOW TO USE THIS TEMPLATE
+# How use the ETL template
+
+## 1. Starting a new repository from this template
 
 > **DO NOT FORK** this is meant to be used from **[Use this template](https://github.com/cloudninedigital/cnd-etl-template/generate)** feature.
 
 1. Click on **[Use this template](https://github.com/cloudninedigital/cnd-etl-template/generate)**
 2. Give a name to your project. See [naming conventions](https://github.com/cloudninedigital)
 3. Wait until the first run of CI finishes  
    (Github Actions will process the template and commit to your new repo)
-4. If you want [codecov](https://about.codecov.io/sign-up/) Reports and Automatic Release to <insert Cloud Nine Digital repo here>  
-  On the new repository `settings->secrets` add your `PIPY_API_TOKEN` and `CODECOV_TOKEN` (get the tokens on respective websites)
-5. Read the file [CONTRIBUTING.md](CONTRIBUTING.md)
-6. Then clone your new project and happy coding!
+4. Then clone your new project and happy coding!
+
+## 2. Clone your project
+
+Follow [these instructions](https://wiki.cloudninedigital.nl/en/Processing-and-Delivery/Software-Development/Git/Getting-started-with-Git#cloning-an-existing-repository-from-a-remote-repository) 
+to clone a repository created on Github.
+
+## 3. Install Terraform and Cloud SDK CLI
```

Comment:
Might be nice to include a brief overview of what Terraform is and what is does (or a link to a description of this). At least it should be clear that Terraform handles the main deployment of the code.